app/types/timesDetails.ts

```diff
@@ -19,13 +19,13 @@ export interface TimesDetails {
         duration: Duration;
         count: number;
     }[];
-    min?: DurationWithStyle;
-    max?: DurationWithStyle;
+    min: DurationWithStyle | null;
```

Comment:
This way is debatable. Let me double check. This will not be a hindrance to merging this PR. I will deal with them when I confirm.

Reply:
https://www.typescriptlang.org/docs/handbook/interfaces.html#optional-properties

I am uncertain about whether it is appropriate to use `optional-properties` in these cases.